data/json/monstergroups/zombie_upgrades.json

```
@@ -38,6 +38,17 @@
       { "monster": "mon_zombie_crawler_pupa_decoy", "freq": 500, "cost_multiplier": 2 }
     ]
   },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_ZOMBIE_BRAINLESS_UPGRADE",
+    "default": "mon_zombie_brainless",
+    "//": "No bionics or fungal",
+    "monsters": [
+      { "monster": "mon_zombie_brainless", "freq": 45, "cost_multiplier": 5 },
+      { "monster": "mon_zombie_ears", "freq": 245, "cost_multiplier": 2 },
+      { "monster": "mon_afs_headless_horror", "freq": 40, "cost_multiplier": 2 }
```

```suggestion
      { "monster": "mon_zombie_brainless", "freq": 45, "cost_multiplier": 2 },
      { "monster": "mon_zombie_ears", "freq": 245, "cost_multiplier": 2 },
      { "monster": "mon_afs_headless_horror", "freq": 40, "cost_multiplier": 5 }
```
Or should the last be mon_afs_headless_zombie instead of horror?